Create homework.html

rawinslow24 opened the pull request:
I ask for user's birthdate in Short Date Format. Then I subtract the birthDay from the currentDate and this gives me the difference in Milliseconds. To convert to days, I use an equation to divide the difference by the product of multiplying 1000 (milliseconds in a second) times 60 (seconds in a minute) times 60 (minutes in an hour) times 24 (hours in a day). I also added a Math.Floor function to round it down to the nearest integer.

rawinslow24 committed:
Create homework.html

I ask for user birthdate in Short Date Format. The I subtract the birthDay from the currentDate and this gives me the difference in Miliseconds. To convert to days, I use an equation to divide it by the product of multiplying 1000 (miliseconds in a second) times 60 (seconds in a minute) times 60 (minutes in an hour) times 24 (hours in a day). I also added a .floor function to round it down to the nearest integer.

rawinslow24 committed:
Update assignment2.html

removed redundant variable

Kilavagora reviewed (commented):
@cp-winslow 
Use **const** everywhere, since you are not re-assigning any variables. 

(1000 * 60 * 60 * 24) does not need parentheses, but it needs a semicolon. BTW, semicolon in JavaScript is optional, but using it could alleviate some potential pitfalls and it does not harm.

rawinslow24 committed:
Update homework.html

Changed var to const on lines 10 & 11, removed parenthesis on line 10, added a semicolon on line 10,  and removed some unnessesary comments that used to be on lines 13 & 14.
